sources/packages/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -135,7 +138,10 @@ export class ApplicationStudentsController extends BaseController {
   @ApiOkResponse({ description: "Application submitted." })
   @ApiUnprocessableEntityResponse({
     description:
-      "Program Year is not active or invalid study dates or selected study start date is not within the program year or APPLICATION_NOT_VALID or INVALID_OPERATION_IN_THE_CURRENT_STATUS or ASSESSMENT_INVALID_OPERATION_IN_THE_CURRENT_STATE.",
+      "Program Year is not active or " +
+      "invalid study dates or selected study start date is not within the program year" +
```

Comment:
The description is becoming big, if there is a way to split them, it would be nice. Not part of your PR, just a thought